docs-article-templates/src/helper/unit-builder.ts

```diff
@@ -41,16 +43,34 @@ export async function createUnits() {
     const unitPath = join(modulePath, `${formattedUnitName}.yml`);
     if (!learnRepoId) {
         learnRepo = repoName;
-    } else {
-        learnRepo = learnRepoId;
     }
+    if (!gitHubID) {
+        author = `...`;
+    }
+    if (!alias) {
+        msAuthor = `...`;
+    }
+
     /* tslint:disable:object-literal-sort-keys */
     const yaml = require("write-yaml");
```

Comment:
Have you tried to use `js-yaml` Im pretty sure `write-yml` uses `js-yaml.dump(obj)` under the hood. Its a more widely used npm package.

Reply:
Hi @bharney-msft , good idea.  Updated.